packages/website/pages/new-file.js

```diff
@@ -176,6 +182,25 @@ export default function NewFile() {
             </div>
           </form>
         </Box>
+        <When condition={error !== ''}>
+          <Alert
+            className="pa4 white"
+            position="top"
+            type="error"
+            timer={4}
+            onTimerEnd={() => setError('')}
```

Comment:
Can this one not auto dismiss please? I want users to retry the upload or explicitly accept the issue and then move on. For large uploads a user may leave the page and then come back and have missed the error message.